### ISSUE TITLE
Cannot find symbol "LocalBroadcastManager"

### ISSUE THREAD

**Issue by MrWeezle** [bsorrentino/cordova-broadcaster#71]
I upgraded my Ionic App to Capacitor 5 and also updated all plugins to the newest compatible version.
I also updated this plugin (cordova-broadcaster) to version 5.0.0 where I saw that a fix for androidx was implemented.
Sadly, the build still fails with the following error message:

```
android/capacitor-cordova-android-plugins/src/main/java/org/bsc/cordova/CDVBroadcaster.java:192: error: cannot find symbol
    return LocalBroadcastManager.getInstance(super.webView.getContext()).sendBroadcast(intent);
           ^
  symbol:   variable LocalBroadcastManager
  location: class CDVBroadcaster`
```
Manually editing the gradle file to add 
`implementation "androidx.localbroadcastmanager:localbroadcastmanager:1.0.0"`
and removing 
`implementation "com.android.support:support-v4:26+"` 
because it is not compatible, results in a working Build in Android Studio and a functioning broadcaster. Leaving the support-library in results in following error messages, thus i removed it:

> Duplicate class android.support.v4.app.INotificationSideChannel found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)
     Duplicate class android.support.v4.app.INotificationSideChannel$Stub found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)
     Duplicate class android.support.v4.app.INotificationSideChannel$Stub$Proxy found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)
     Duplicate class android.support.v4.os.IResultReceiver found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)
     Duplicate class android.support.v4.os.IResultReceiver$Stub found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)
     Duplicate class android.support.v4.os.IResultReceiver$Stub$Proxy found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)
     Duplicate class android.support.v4.os.ResultReceiver found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)
     Duplicate class android.support.v4.os.ResultReceiver$1 found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)
     Duplicate class android.support.v4.os.ResultReceiver$MyResultReceiver found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)
     Duplicate class android.support.v4.os.ResultReceiver$MyRunnable found in modules core-1.10.0-runtime (androidx.core:core:1.10.0) and support-compat-26.1.0-runtime (com.android.support:support-compat:26.1.0)

But building it with Jenkins always fails, because the gradle file is overridden with capacitor sync.

The import in CSVBroadcaster.java is 
`import androidx.localbroadcastmanager.content.LocalBroadcastManager;`
which is different from the value I saw in the PR a few days ago.

Is there any solution to get a working build with Jenkins?

**Comment by bsorrentino** [comment]
Hi @MrWeezle 

I merged PR without any modification and the [last PR commit](https://github.com/bsorrentino/cordova-broadcaster/pull/70/commits/79e9ba97624ce503b7fcd485da914260171d1ee7) is: 

```java
- import androidx.LocalBroadcastManager;
+ import androidx.localbroadcastmanager.content.LocalBroadcastManager;
```

This is mostly a capacitor issue and you need further investigations on it. When I'll have a time-slot give I try to figure out the problem

**Comment by MrWeezle** [comment]
I found a solution that works for me:

In the plugin.xml file in the section for android I changed 

`<framework src="com.android.support:support-v4:26+" />`
to
`<framework src="androidx.localbroadcastmanager:localbroadcastmanager:1.0.0" />`

The Build process updates the gradle file correctly and no errors are shown.
For validation with a Jenkins Build I forked this project, implemented the change and ran a build. No errors.

I cannot confirm that changing the used library in the framework part works for every use case, but it works for our app.
